db/migrations/bcda/000003_aco_blacklist.down.sql

```diff
@@ -0,0 +1,3 @@
+BEGIN;
```

Comment:
Could maybe use a comment explaining what is being dropped. Not really needed here since the migration is so small but would be a good practice for some of the bigger ones